lib/src/lib.rs

```diff
@@ -74,6 +74,12 @@ impl Color {
             Color::Black => Color::White,
         }
     }
+    fn get_index(&self) -> usize {
+        match self {
+            Color::White => 0,
+            Color::Black => 1,
+        }
+    }
```

Comment:
Use #[repr(u8)] on Color instead of implementing method
https://doc.rust-lang.org/reference/items/enumerations.html#custom-discriminant-values-for-fieldless-enumerations